certification/internal/k8s/deployable_by_olm.go

```diff
@@ -0,0 +1,209 @@
+package k8s
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"time"
+
+	operatorv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/errors"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/cli"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+	yaml "gopkg.in/yaml.v2"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/client-go/rest"
+
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+const (
+	catalogSourceNs = "openshift-marketplace"
+)
+
+var (
+	k8sconfig                     *rest.Config
+	err                           error
+	subscriptionName, packageName string
+	channel, ooNamespace          string
+	operatorGroup, catalogSource  string
+	catalogImage                  string
+	targetNamespaces              []string = []string{ooNamespace}
+)
```

Comment:
Do these variables need to be package-level variables? 

I see you're setting things up in an initializer function, can we instead make these struct-level variables?